src/app/containers/ChatListItem/index.tsx

```diff
@@ -29,7 +30,7 @@ class ChatListItemContainer extends React.Component<IProps & IFuncs & IState, an
   public renderItem = () => {
     const { id, selected, peer, peerData, history } = this.props;
     const shortName = getPeerShortName(peer, peerData);
-    const lastMsg = getLastMessage(history);
+    const lastMsg = history.byId[this.props.dialog.top_message] || {message: ''};
```

Comment:
For react shallow comparation default values (as {message: ''}) should be created _outside_ of any render, because {message: ''}!=={message: ''}